go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/weaviate/weaviate-go-client/v5
 
-go 1.22.7
+go 1.24
+
+toolchain go1.24.1
```

Comment:
do we need to change this? I think that would force our users to update

Reply:
I just thought the same and reverted these changes 👍